services/admin/schema.sql

```diff
@@ -20,12 +28,16 @@ create table cvrs (
 
 create table cvr_files (
 	id varchar(36) primary key,
+	election_id varchar(36) not null,
 	signature text,
 	filename text,
 	timestamp text,
 	imported_cvr_count integer,
 	duplicated_cvr_count integer,
 	scanner_ids text,
 	precinct_ids text,
-	contains_test_mode_cvrs boolean
+	contains_test_mode_cvrs boolean,
+	foreign key (election_id) references elections(id)
+	  on update cascade
```

Comment:
what does `on update cascade` do here?

Reply:
Were the election ID to change, I think this means that update would cascade to the `election_id` values that reference it.

Reply:
Interesting. Doesn't seem like something we need to support but doesn't seem to cause any harm